auth_plugin_pyauth.c

```diff
@@ -69,9 +69,25 @@ static PyObject *pyauth_mosquitto_topic_matches_sub(PyObject *self unused, PyObj
     return PyBool_FromLong(res);
 }
 
+static PyObject *pyauth_mosquitto_log_printf(PyObject *self unused, PyObject *args)
+{
+    int loglevel;
+    char *fmt;
+
+    if (!PyArg_ParseTuple(args, "is", &loglevel, &fmt))
+    return NULL;
+
+    mosquitto_log_printf(loglevel, fmt);
```

Comment:
Something like `mosquitto_log_printf(loglevel, "%s", msg);` is better, as we won't risk evaluating printf syntax in log message.

Reply:
done